client/src/components/ReviewList.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Review from './Review.jsx';
+import styled from 'styled-components'
```

Comment:
Do you need to import this?

Reply:
Not really, just dumped it in all the jsx files. Just removed it from the files that don't need it.